habitat-lab/habitat/utils/env_utils.py

```diff
@@ -14,7 +14,9 @@
 
 
 def make_env_fn(
-    config: "DictConfig", env_class: Union[Type[Env], Type[RLEnv]]
+    config: "DictConfig",
+    env_class: Union[Type[Env], Type[RLEnv]],
+    dataset=None,
```

Comment:
Why does the make env function need to have a dataset specified?

Reply:
This is to allow passing a dataset to the environment setup so it's not recreated every time. However, by default, it is recreated and the default behavior is unchanged from what was there before.